.gitignore

```diff
@@ -9,4 +9,6 @@
 /uploads/
 node_modules/
 wordpress-coding-standards
-php-codesniffer
\ No newline at end of file
+php-codesniffer
+plugins/wordpress-importer/
+php.ini
```

Comment:
They can be there, specifically to override the root php.ini file. But I see what you mean. I removed it.